static/src/javascripts/projects/common/modules/adfree-thrasher-simple.js

```diff
@@ -0,0 +1,50 @@
+define([
+    'bean',
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/template',
+    'common/views/svgs',
+    'text!common/views/commercial/adfree-thrasher-simple.html'
+], function (
+    bean,
+    fastdom,
+    $,
+    _,
+    template,
+    svgs,
+    adfreeThrasher
+) {
+
+    /**
+     * Message which is shown as an trash component to encourage people in joining membership.
+     *
+     * @constructor
+     * @param {Object=} options
+     */
+    var AdfreeThrasher = function (options) {
+        var opts = options || {};
+        this.$container = opts.$container || '';
```

Comment:
This default operator suggests that we can pass the constructor an undefined $container, but if that happens, it seems that `write` will try calling .after on an empty string, and throw an exception.